src/components/applications/children.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div v-if="campApps.length > 0" class="container-fluid mx-0">
+        <h4 class="text-left gray font-weight-bold">Children who may attend camp</h4>
+        <table class="table">
+            <thead>
+                <tr>
+                    <th colspan="2" class="text-center"></th>
+                    <th colspan="2" class="text-center">Last Application</th>
+                </tr>
+                <tr>
+                    <th class="text-center">Name</th>
+                    <th class="text-center">Age</th>
+                    <th class="text-center">Submitted</th>
+                    <th class="text-center">Status</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="child in children" v-bind:key="child[0].childName">
+                    <td>{{child[0].childName}}</td>
+                    <td>{{child[0].age}}</td>
+                    <td>
+                        <div v-for="app in child" v-bind:key="app._id.$oid">
+                            <router-link :to="{ name: 'applicationSubmitted', params: { id: app._id.$oid } }">
+                            {{app.date_signed}}
+                            </router-link>
+                        </div>
+                    </td>
+                    <td>
+                        <div v-for="app in child" v-bind:key="app._id.$oid">
+                            {{app.status}}
+                        </div>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</template>
+<script>
+import localforage from "../../sessionUtils";
+import axios from "axios";
+import _ from 'lodash';
+export default {
+    name: "appChildren",
+    data() {
+        return {
+            sessionId: '',
+            applications: [],
+            campApps: [],
+            children: [],
+        }
+    },
+    created: function() {
+        localforage.getItem("X_TOKEN")
+        .then(session => {
+            this.sessionId = session;
+            axios.get("/api/v1/profile/" + session + "/applications", {
```

Comment:
Hey, I altered the back-end pretty significantly. I implemented your changes from your last PR, but I tweaked some pieces of it. You can replace the url here with this: `"/api/v1/profile/applications"` and it should return the same thing.